### DISCUSSION TITLE
SPLITTER

### DISCUSSION THREAD

**Discussion by ted2000** [Velociraptor115/DSPMods#13] [General]
**Mod**
Splitter s

**Describe the bug**
How to update splitter so that if you put 2 or more on top of each other that they work like a storage bin but with no storage ?

**To Reproduce**
1. Go to '...' Place splitter on ground
2. Click on '....'Place second splitter on top
3. Do '....'connect conveyer belt input to either splitter and the other conveyer belt out put to the other splitter as if they were bins
4. See error Currently will not work and I have just installed Unity and am a novice at it and I would like to understand the concepts on behaver linkage aspects in the dsp program of the STL ANNAMATION and INTERGRATION protocols in making of mods ?

**Expected behavior**
connect conveyer belt input to either splitter and the other conveyer belt out put to the other splitter as if they were bins

**Screenshots**

**Environment (please complete the following information):**
 - OS version: [e.g. Windows 10 B]
 - Game version: [latest]
 - List of other installed mods

**Additional context**

**Comment by Velociraptor115** [comment]
Hello @ted2000

The idea is a nice one, although it will involve a lot of discussion on how it could be done

If you are available on Discord, please join the [DSP server](https://discord.gg/dysonsphereprogram) and/or the [DSP modding server](https://discord.gg/DKqBwYJd)
Otherwise, we can continue the discussion here

If you haven't made a sample mod for DSP yet, you can start by following the steps outlined [here](https://wiki.customizing.space/en/Guides/raptor-getting-started)
Feel free to ask questions if anything is not clear

**Reply by ted2000** [reply]
Thank you
